.gitignore

```diff
@@ -15,4 +15,5 @@ proposal/proposal.bbl
 proposal/proposal.blg
 simulations/logs/*
 simulations/*.toolbox
-venv
\ No newline at end of file
+venv
+.idea
```

Comment:
We can leave this in your local .git/info/exclude. The entire repo can avoid having ignores for all contributors editor dotfiles.